README.md

```diff
@@ -1,10 +1,11 @@
 # Pistache
 
 [![Travis Build Status](https://travis-ci.org/oktal/pistache.svg?branch=master)](https://travis-ci.org/oktal/pistache)
+[![Coverage Stagus](https://codecov.io/gh/oktal/pistache/branch/master/graph/badge.svg)](https://codecov.io/gh/oktal/pistache)
 
 Pistache is a modern and elegant HTTP and REST framework for C++.
 
-It is entirely written in pure-C++11 and provides a clear and pleasant API
+It is entirely written in pure-C++14 and provides a clear and pleasant API
```

Comment:
To be honest, we still don't use C++ 14.

Reply:
Maybe say something about min requirement of c++11, but "soon" to be c++17?